docs/_posts/2020-05-11-release-0-8.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Announcing Pomerium 0.8
+date: 2020-5-11
+tags:
+  - release
+  - pomerium
+  - announcement
+author: Bobby DeSimone
+---
+
+# Announcing Pomerium 0.8
+
+We are excited to announce the [0.8 release] of Pomerium which adds support for some of our most requested features including:
+
+- [**Automatic Certificate Management**] — Pomerium can now be configured to automatically retrieve and renew certificates, adding HTTPS to all Pomerium managed routes. In addition, Pomerium will do [OCSP stapling](https://en.wikipedia.org/wiki/OCSP_stapling) for automatic and custom certificates alike.
+- [**Advanced Route Matching**] — Operators can now write access policy that supports route matching based on [regex], [prefix], and [path] settings. Pomerium now has the flexibility to support multiple and layered authorization policies across a single managed route.
+- And finally, this release adds [**Github**](https://github.com/) as a supported identity provider.
+
+Pomerium had 95 commits from 8 authors across 5 organizations in this release. This release also includes additional new features, general improvements, and bug fixes, a complete list of which can be found in the [changelog].
+
+As always, we recommend upgrading and testing this release in an isolated environment. If you experience any issues, please report them on the Pomerium GitHub [issue tracker].
+
+<SimpleNewsletter/>
```

Comment:
Is this tag meant to be here?

Reply:
Yes, it adds the newsletter at the bottom :)

Reply:
👍